src/management-system-v2/components/layout.tsx

```diff
@@ -62,53 +62,54 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
             </Link>
           </div>
           {loggedIn ? (
-            <Menu theme="light" mode="inline" selectedKeys={[activeSegment]}>
-              {ability.can('view', 'Process') || ability.can('view', 'Template') ? (
-                <>
-                  <ProcessSider></ProcessSider>
-                  <Divider />
-                </>
-              ) : null}
+            <SiderMenu />
+            // <Menu theme="light" mode="inline" selectedKeys={[activeSegment]}>
+            //   {ability.can('view', 'Process') || ability.can('view', 'Template') ? (
+            //     <>
+            //       <ProcessSider></ProcessSider>
+            //       <Divider />
+            //     </>
+            //   ) : null}
```

Comment:
If you refactor / change parts of the code that are not just temporary, you can confidently remove those and don't need to comment everything out.
We can always go back in the commit history to get removed code back :)

Reply:
Noted! Thanks a lot! :)